### PR TITLE
Fix the width of the plotting status bar for Conda builds

### LINKED ISSUE #33732: Plot window status bar changes size
**Describe the bug**
The status bar at the bottom of a 2D plot window (e.g. contour or colorfill) changes size when the mouse is moved over the plot image or the z axis image. This is likely due to the matplotlib 3.5 upgrade.

**To Reproduce**
1. Load a workspace, e.g. MAR11060 from the training course data.
2. Open a colorfill plot by right clicking and selecting Plot->Colorfill.
3. Move the mouse over the plot image and off again, you will see that the GUI moves around because the status bar changes size.

**Expected behavior**
Status bar text should be on one line, as in Mantid Workbench version 6.3.0.

**Screenshots**
image

**Platform/Version (please complete the following information):**
 - OS: Conda environment with matplotlib 3.5 or newer
 - Mantid Version: main branch

### PR DESCRIPTION
**Pull request by sf1919**
**Description of work.**
Conda builds use version 3.5 of matplotlib or higher. This version changes how the x and y coords are shown on the status bar of plots. Unfotunately it is not possible to get the information to show on a single line (this seems to be something specific with matplotlib). Instead the status bar width has now been fixed for those using matplotlib 3.5 or higher to prevent the constant resizing of the bar as you move on and off the plot image or the z axis bar.

**To test:**
Using a Conda installation:

1. Load a workspace, e.g. MAR11060 from the training course data.
2. Open a colorfill plot by right clicking and selecting Plot->Colorfill.
3. Move the mouse over the plot image and off again. The GUI no longer moves around because the status bar height is static.

Fixes #33732 . 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Review by cailafinn — CHANGES_REQUESTED** [review]
Just a typo, otherwise fix works well, no longer jumping around

**Review by cailafinn — COMMENTED** [review]
Code looks good to me now. As discussed offline I think the fix for the status appearing over two lines could probably go on this PR.

> It’s in mantid_navigation_toolbar.py L129
> The new output just has a \n character in it so that just needs stripping out and it’ll go back to being on one line
> So it might be worth adding to this PR I’ve been reviewing. You’d just need a new method in place of self.statusbar_label.setText() on L214 that would take a string, strip out the newlines and then set the text on the statusbar label

File: qt/applications/workbench/workbench/plotting/figuremanager.py [hunk 1]
```diff
@@ -242,9 +242,9 @@ def __init__(self, canvas, num):
         cs = canvas.sizeHint()
         sbs = self.window.statusBar().sizeHint()
         self._status_and_tool_height = tbs_height + sbs.height()
+
         height = cs.height() + self._status_and_tool_height
         self.window.resize(cs.width(), height)
-
```
Comment (ConorMFinn) [inline qt/applications/workbench/workbench/plotting/figuremanager.py:247]: Is it worth putting this back to how it was just to keep the git history nice?

**Review by cailafinn — CHANGES_REQUESTED** [review]
One really tiny nitpick, but this looks good and works well

**Review by cailafinn — APPROVED** [review]
A really neat fix, nice work 👍

**Pull request merged**